app/src/main/java/org/mozilla/fenix/settings/SupportUtils.kt

```diff
@@ -24,6 +24,7 @@ object SupportUtils {
     const val POCKET_TRENDING_URL = "https://getpocket.com/fenix-top-articles"
     const val WIKIPEDIA_URL = "https://www.wikipedia.org/"
     const val FENIX_PLAY_STORE_URL = "https://play.google.com/store/apps/details?id=${BuildConfig.APPLICATION_ID}"
+    const val STUDIES_LEARN_MORE = "https://support.mozilla.org/en-US/kb/shield?as=u&utm_source=inproduct"
```

Comment:
You don't need to add the whole URL here 😁 You can just take the last part of it ("shield") and add it below in `SumoTopic`. That way it gets localized correctly instead of "en-US"

Reply:
```
SupportUtils.getSumoURLForTopic(SupportUtils.SumoTopic.WHATS_NEW)
```

Reply:
Yeah, I was a place holder until, we get the final Android version :)

Reply:
I updated it with the final final sumo topic :)